src/libmongoc/tests/test-mongoc-bulkwrite.c

```diff
@@ -677,4 +714,12 @@ test_bulkwrite_install (TestSuite *suite)
                       test_framework_skip_if_max_wire_version_less_than_25, // require server 8.0
                       test_framework_skip_if_mongos // Creating 100k collections is very slow (~5 minutes) on mongos.
    );
+
+   TestSuite_AddFull (suite,
+                      "/bulkwrite/execute_requires_client",
+                      test_bulkwrite_execute_requires_client,
+                      NULL /* dtor */,
+                      NULL /* ctx */,
+                      test_framework_skip_if_max_wire_version_less_than_25, // require server 8.0
```

Comment:
```suggestion
                      test_framework_skip_if_max_wire_version_less_than_25 // require server 8.0
```

Remove trailing comma to fix [error](https://spruce.mongodb.com/task/mongo_c_driver_sanitizers_matrix_asan_asan_sasl_cyrus_openssl_ubuntu2004_clang_compile_patch_8eba3f6f52bff67389124eef36fe120dc6fef15f_675074945d190d0007e61a6d_24_12_04_15_26_14/logs?execution=0):
```
../../../src/libmongoc/tests/test-mongoc-bulkwrite.c:718:4: error: expected expression
   TestSuite_AddFull (suite,
  ^
```

Reply:
Sorry for not making this a draft PR. I was actually relying on Evergreen to build it since it was difficult to do so directly from my submodule (as opposed to https://github.com/mongodb/mongo-c-driver/pull/1803, which I did through a separate fork).